grasa_event_locator/views.py

```diff
@@ -187,7 +187,7 @@ def event(request, eventID):
 
 def login(request):
         if request.user.is_authenticated:
-                return HttpResponseRedirect("index.php")
+                return HttpResponseRedirect("search/search.html")
```

Comment:
This change makes sense to me, but we should use the `reverse()` method to dynamically set the URL to the internal page name, so if the URI ever changes in the future, Django will automatically handle that. Get a deeper explanation of what I mean in [this blog post](https://overiq.com/django-1-10/redirecting-urls-in-django/):

`Instead of manually typing the URLs, we can also use reverse() method with HttpResponseRedirect to automatically create urls.`

This needs to be tested, but I think the fix would be `reverse(haystack_search)` for the parameter here, based on the `urls.py` file:

https://github.com/jwflory/django-rit-grasa/blob/c0817411f626378fb77e7cbcf381aaf1c7a6a89e/grasa_event_locator/urls.py#L37